archetype/icefaces-portlet/src/main/resources/archetype-resources/build.gradle

```diff
@@ -14,9 +14,6 @@ dependencies {
     }
     compile group: 'org.icefaces', name: 'icefaces-ace', version:'3.3.0'
     compile group: 'org.icefaces', name: 'icefaces-compat', version:'3.3.0'
-    compile('org.icepush:icefaces:3.3.0') {
```

Comment:
@vsingleton Similar to my comment in #92, is this related to the alloy version snapshot problem?